src/EventHandler/Listener/StartSpanListener.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
This file must not be part of the final PR, it looks like testing/evaluation code. We 100% do not want this random attribute added to all spans

Reply:
Yeah, makes sense. Where do we include the listeners as an example?

Reply:
Just create some inline functions in the example php file - I gave an example of doing that in a comment the other day.

Reply:
As, I mentioned above wouldn't it couple listener with dispatcher?

Reply:
Given that the dispatcher will accept any callable as an event handler, I think that is very loose coupling. Are we talking about the same thing?